client/gutenberg/editor/controller.js

```diff
@@ -323,5 +299,6 @@ export const redirectSiteEditor = async ( context, next ) => {
 	// We aren't using `getSiteEditorUrl` because it still thinks we should gutenframe the Site Editor.
 	const siteAdminUrl = getSiteAdminUrl( state, siteId );
 	const siteEditorUrl = addQueryArgs( queryArgs, `${ siteAdminUrl }site-editor.php` );
-	return location.assign( siteEditorUrl );
```

Comment:
The site editor with the iframe has lots of query args for different features, we have to keep those args to avoid any potential side effects. See https://github.com/Automattic/wp-calypso/blob/9a4894f73c79f932bf97b0590a588b652f6eb8d2/client/gutenberg/editor/calypsoify-iframe.tsx#L809.

Reply:
I made a [PR](https://github.com/Automattic/wp-calypso/pull/76072) to address this issue